.travis.yml

```diff
@@ -68,7 +68,7 @@ script:
   - COVERAGE_PROCESS_START="$PWD/.coveragerc" COVERAGE_FILE="$PWD/.coverage" PYTHONPATH="$PWD" sct_testing -d 1
   - coverage combine
   - ./unit_testing.sh
-  - bash -c "pip install sphinx sphinxcontrib.programoutput; cd $HOME/build/neuropoly/spinalcordtoolbox/documentation/sphinx; make html; echo"
+  - bash -c "pip install sphinx sphinxcontrib.programoutput sphinx_rtd_theme; cd $HOME/build/neuropoly/spinalcordtoolbox/documentation/sphinx; make html; echo"
```

Comment:
you can remove this commit and rebase, you'll see that this change has been already merged